### PR TITLE
YAML ace editor fix

### PR DESCRIPTION
**Pull request by rhamilto**
Fixes bug where ace-editor for yaml didn't have a height inside of an overlay/panel

### CODE REVIEW

File: app/styles/_editor.less [hunk 1]
```diff
@@ -50,7 +50,9 @@
   text-align: right;
 }
 
-#from-file .editor {
-  height: 350px;
+#from-file,
+#yaml-file {
```
Comment (spadgett) [inline app/styles/_editor.less]: It looks like this rule with apply to Edit YAML action, not just Import YAML? Edit YAML currently grows with the page.

**Comment by rhamilto** [comment]
> It looks like this rule with apply to Edit YAML action, not just Import YAML? Edit YAML currently grows with the page.

Yeah.  And Add to Project > From File also still has no height.  :/  Will fix afternoon.

**Comment by rhamilto** [comment]
This should do it, @spadgett

**Comment by spadgett** [comment]
Thanks @rhamilto 

[merge][severity: blocker]

**Pull request merged**